app/components/BuynBTC.tsx

```diff
@@ -0,0 +1,72 @@
+import { Card, CardContent } from "./ui/card";
+import { Button } from "./ui/button";
+import { FormInput } from "./form/FormInput";
+import { FormProvider, useForm } from "react-hook-form";
+import { useContext } from "react";
+import { WalletContext } from "~/providers/ByieldWalletProvider";
+import { ByieldWallet } from "~/types";
+import { SuiModal } from "./Wallet/SuiWallet/SuiModal";
+
+interface ExchangeRateProps {
+	fee: number;
+	youReceive: number;
+}
+
+function Fee({ youReceive }: ExchangeRateProps) {
+	return (
+		<Card className="p-4 bg-azure-10 rounded-2xl h-14">
```

Comment:
let's define constants for those colors and inject them. If we will need to change slightly a color, we do it in one place , rahter than in multiple places. something like:
```
// file: colors.ts
export const cardBG = "bg-azure-10"

// font colors
expot const fontDefault = ...
```

Reply:
please make a task to update in a separate PR other places.

Reply:
no no. this is defined in the tailwind config file already. It will be reflected everywhere automatically done once config is updated.

Reply:
ok, thanks